engines/bops_enforcements/app/views/bops_enforcements/assign_users/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :page_title do %>
+  Assign case record - <%= t("page_title") %>
+<% end %>
+
+<% content_for :title, "Assign case record" %>
+
```

Comment:
Would these lines work moved into the enforcement partial? Would be nice to avoid having to add too much for each new page.